int8 direction # only rotate 1 = clockwise (right?); -1 = counter-clockwise (left?)
float32 angular_speed         # m/s    (must be >Â 0)
---
bool success